.changelog/3355.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:improvement
+core: update swagger and gateway yml to include jobs and ui endpoints.
+
```

Comment:
Hey! Noting that we typically reserve changelog entries for user facing bugs/features/improvements. This is more of a dev thing, so no need for a changelog 👍🏻

Reply:
OH ok! Hmmm i added this because the build was failing on the changelog check? Is there something I need to do to make it skip this step in the build?

Reply:
Yep! We run that check on all PRs, so you can opt out of it for ones that don't need it with `pr/no-changelog`.

Reply:
@teresamychu if it's a "no changelog" kind of PR, you can just add the `pr/no-changelog` label! 😄

Reply:
ooooo ok got it! I'll do that now and remove this file. Thanks guys!